<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>951</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QWidget" name="Menu">
     <widget class="QLabel" name="title_menu">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome to the multi-purpose image editing program</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="menu_header">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose the editing mode or filter:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="black_and_white_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Black and white filter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="flip_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Flip Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="purple_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply purple filter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="old_tv_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Old TV filter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="crop_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Crop Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resize_button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Resize Image</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="black_and_white">
     <widget class="QLabel" name="black_and_white_menu">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Black and White filter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="black_and_white_label1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter the path of the image:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="black_and_white_filename">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>661</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="black_and_white_error_message">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="black_and_white_savenew">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as new image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="black_and_white_savesame">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as same image</string>
      </property>
     </widget>
     <widget class="QLabel" name="black_and_white_savesuccessful">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Image Saved Successfully. Returning to menu.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="flip">
     <widget class="QLabel" name="flip_menu">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Flip Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="flip_label1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter the path of the image:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="flip_filename">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>601</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="flip_errormessage">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="flip_V">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>93</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertically</string>
      </property>
     </widget>
     <widget class="QPushButton" name="flip_H">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>93</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Horizontally</string>
      </property>
     </widget>
     <widget class="QLabel" name="flip_label2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Do you want to flip the image vertically or horizontally?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="flip_savenew">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>230</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as new image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="flip_savesame">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>300</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Save to same name and path</string>
      </property>
     </widget>
     <widget class="QLabel" name="flip_savesuccessful">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>420</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Image saved successfully. Returning to menu...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="crop"/>
    <widget class="QWidget" name="resize"/>
    <widget class="QWidget" name="purple"/>
    <widget class="QWidget" name="old_tv"/>
    <widget class="QWidget" name="newsave">
     <widget class="QLabel" name="newsave_menu">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New Save</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="newsave_label1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose directory then enter the new name of the file:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="newsave_newfilename">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>601</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="newsave_browse">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>110</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="newsave_savesuccessuful">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>370</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Image saved successfully. Returning to menu.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="newsave_errormessage">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
